base_agent/loco_mc_agent.py

```diff
@@ -60,6 +63,10 @@ def init_event_handlers(self):
         logging.info("creating all tables for Visual programming and error annotation ...")
         create_all_tables(self.conn)
 
+        @sio.on("getMemoryDB")
```

Comment:
We only need to do this when frontend sends a socket event called "getMemoryDB" (think of this as frontend asking for this).
Because we just want to push out this info from backend without frontend explicitly asking for it, I think we can just put the :
```
sio.emit("memoryState", self.agent.dashboard_memory["db"])
```
here: https://github.com/facebookresearch/droidlet/pull/24/files#diff-43d16fafcfac417f5667bb9a8af7cd9f1ab30a4fdfef697adaefd07fde5a21a3R277